scripts/gha/build_testapps.py

```diff
@@ -400,15 +414,15 @@ def _generate_makefiles_from_repo(sdk_dir):
 # build required ios frameworks based on makefiles
 def _build_ios_framework_from_repo(sdk_dir, api_config):
   ios_framework_builder = os.path.join(
-    sdk_dir, "build_scripts", "ios", "build.sh")
-  
+      sdk_dir, "build_scripts", "ios", "build.sh")
+
   # build only required targets to save time
   target = set()
   for framework in api_config.frameworks:
     target.add(os.path.splitext(framework)[0])
```

Comment:
Will be nice to see an example string in comments of what is expected to happen. I had to follow the calls to see an example of what values could possibly come as framework.

Reply:
Added a comment "firebase_analytics.framework -> firebase_analytics", let me know if that's not what you had in mind.